File: tests/fixtures/springboot-comprehensive/src/main/java/com/example/orders/Order.java
package com.example.orders;

import jakarta.validation.constraints.*;

public class Order {
    @NotBlank
    private String customerName;

    @NotBlank
    @Email
    private String customerEmail;

    @NotNull
    @Positive
    private Integer amount;

    @NotBlank
    @Pattern(regexp = "EUR|USD|GBP")
    private String currency;

    public String getCustomerName() { return customerName; }
    public void setCustomerName(String customerName) { this.customerName = customerName; }
    public String getCustomerEmail() { return customerEmail; }
    public void setCustomerEmail(String customerEmail) { this.customerEmail = customerEmail; }
    public Integer getAmount() { return amount; }
    public void setAmount(Integer amount) { this.amount = amount; }
    public String getCurrency() { return currency; }
    public void setCurrency(String currency) { this.currency = currency; }
}

File: tests/fixtures/aspnet-comprehensive/Models/Order.cs
using System.ComponentModel.DataAnnotations;

namespace AspNetComprehensive.Models;

public class Order
{
    [Required]
    public string CustomerName { get; set; } = "";

    [Required]
    [EmailAddress]
    public string CustomerEmail { get; set; } = "";

    [Required]
    [Range(1, int.MaxValue)]
    public int Amount { get; set; }

    [Required]
    [RegularExpression("^(EUR|USD|GBP)$")]
    public string Currency { get; set; } = "EUR";
}